usr/share/rear/finalize/Linux-i386/670_run_efibootmgr.sh

```diff
@@ -32,7 +32,10 @@ fi
 
 # Skip if there is no esp_mountpoint directory (e.g. the fallback ESP mountpoint may not exist).
 # Double quotes are mandatory here because 'test -d' without any (possibly empty) argument results true:
-test -d "$esp_mountpoint" || return 0
+if ! test -d "$esp_mountpoint" ; then
+    LogPrint "Skip creating EFI Boot Manager entries (ESP mountpoint $esp_mountpoint does not exist)"
```

Comment:
Nitpicking:
Strictly speaking $esp_mountpoint may exist but is no directory.

Reply:
I changed it to
"(no ESP mountpoint directory $esp_mountpoint)"